conf.yaml

```diff
@@ -71,7 +71,7 @@ contents_title:     Elastic Stack and Product Documentation
 #   <key>: &<variable> <value>
 # The keys don't really matter, but by convention the are the same as the variable.
 variables:
-  stackcurrent: &stackcurrent 8.0
+  stackcurrent: &stackcurrent 8.1
   stacklive: &stacklive [ master, 8.1, 8.0, 7.17 ]
 
   stacklivemain: &stacklivemain [ main, 8.1, 8.0, 7.17 ]
```

Comment:
Should 8.1 be removed from `stacklive` and `stacklivemain`?

Reply:
Oh probably! Good catch :)